core-primitives/enclave-api/src/remote_attestation.rs

```diff
@@ -441,6 +441,47 @@ impl RemoteAttestationCallBacks for Enclave {
 
 		ensure!(ret == sgx_status_t::SGX_SUCCESS, Error::Sgx(ret));
 
+		unsafe {
+			trace!("FMSPC: {:?}", hex::encode(fmspc));
```

Comment:
I believe `hex` import is missing for this.
Also the `FMSPC` is not available in this scope.

Reply:
ya just ported code over forgot the toml changes